shared/telemetry/README.md

```diff
@@ -0,0 +1,121 @@
+# Welcome to the Telemetry Module
+
+This module is used to collect telemetry data from the node.
+
+At the moment, we are using two types of metrics:
+
+- Time series metrics: collected periodically and are stored in a database.
+- Event series metrics: events that occur and are tracked and stored in a database.
+
+
+At the moment, we are using:
+- Prometheus: for timeseries
+- Plain Logs: for event metrics (_Might be subsituted in the future with an events database_)
+
+
+# Usage
+
+## Node Configuration
+
+It is necessary to provide a telemetry configuration to your node:
+
+```json
+ "enable_telemetry": true,
+ "telemetry": {
+    "address": "0.0.0.0:9000",
+    "endpoint": "/metrics"
+  }
+```
+
+`enable_telemetry`: is a boolean json entry defined at the root of the document that tells the node whether to use the telemetry module or use a NOOP version.
+`address`: is the prometheus server's address that the telemetry module will listen on.
+`endpoint`: the scraping endpoint that prometheus exposes through the telemetry module.
```

Comment:
Why are we using the word `scraping` here?

Reply:
Telemetry nomenclature. If you take a peak at the `victoria-metrics` configuration, you will find that the configuration key to hook it up to prometheus (_for instance_) is called: `scrape:`...

I've changed this comment to avoid the confusion.